Makefile

```diff
@@ -3,4 +3,4 @@ test: lint
 
 lint:
 	flake8 --exclude .git,__pycache__,lib/certifi,lib/idna,lib/psycopg2,lib/urllib3,lib/chardet,lib/httplib2,lib/requests,lib/yaml
-	black --check --diff .
+	# black --check --diff .
```

Comment:
a) Commented code
b) If we're disabling black because of the python version changes, can we figure out how to get Black working under e.g. python 3.6, whilst still running our tests with 3.4?

Reply:
this is a temporary situation. IMHO we should ASAP migrate to a higher version of python. 

I'm not interested in creating bandaid, but if you want to check black under 3.6 and run code under 3.4 than go ahead :)

Reply:
I also think that we should migrate to a higher version of Python, but that's not something that our team has control over.

I think that we should indeed perform black checks under python 3.6, and run the code under python 3.4.

Some quick googling found this link, which suggests that this is fairly trivial: https://stackoverflow.com/a/20621143/2230226

Can you please implement this? I really don't want us to lose black in the buildpack, as I think that it adds a lot of readability, and enforcing this in CI is a fairly elegant way to do things.

Reply:
It might be worth looking into the build stages functionality of Travis: https://docs.travis-ci.com/user/build-stages/

Reply:
That looks even better!

Reply:
PR into this one: https://github.com/mendix/cf-mendix-buildpack/pull/267